airbyte-commons/src/main/java/io/airbyte/commons/version/AirbyteVersion.java

```diff
@@ -69,14 +72,18 @@ public String getPatchVersion() {
     return patch;
   }
 
+  @Override
+  public int compareTo(final AirbyteVersion another) {
```

Comment:
since there are at least two reasonable ways to compare AirbyteVersions to each other I am preferring to stop implementing compareTo and instead add methods with explicit names to do the different types of comparison.

Reply:
Shouldn't you remove the `implements` in that case?

Reply:
yes, tahnks